__tests__/routes/locations.test.js

```diff
@@ -25,79 +35,120 @@ describe('locations router endpoints', () => {
       const res = await request(server).get('/locations');
       expect(res.status).toBe(200);
       expect(res.body.data.length).toBe(0);
+
+      // Verify that the mock was called
+      expect(Locations.findAll.mock.calls.length).toBe(1);
     });
   });
 
   describe('the GET locations/:groomerId route', () => {
     it('should return a 200 code when it finds the id', async () => {
-      Locations.findBy.mockResolvedValue([
-        {
-          id: '1',
-          groomerId: 'bftqzfqkq3xzj953q64r',
-          businessName: "Gillian's Fine Pet Grooming",
-          address: '032 Gordon Creek Apt. 557',
-          email: 'Carolanne.Ullrich14@gmail.com',
-          phoneNumber: '414-962-7162',
-          lat: -53.91525,
-          lng: -96.60353,
-        },
-      ]);
-      const res = await request(server).get('/locations/bftqzfqkq3xzj953q64r');
+      const location = generate.buildLocation();
+      Locations.findBy.mockResolvedValue([location]);
+      const res = await request(server).get(`/locations/${location.groomerId}`);
```

Comment:
Mocking using the object factory functions look **_much_** cleaner!

Reply:
Way to condense the code so it doesn't take look so bloated